packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -621,6 +621,22 @@ new VpcEndpointService(this, 'EndpointService', {
 });
 ```
 
+Endpoint services support private DNS, which makes it easier for clients to connect to your service by automatically setting up DNS in their VPC.
+You can enable private DNS on an endpoint service like so:
+
+```ts
+import { VpcEndpointServiceDomainName } from '@aws-cdk/aws-route53';
+
+new VpcEndpointServiceDomainName(stack, 'EndpointDomain', {
+  endpointService: vpces,
+  domainName: 'my-stuff.aws-cdk.dev',
+  publicZone: zone,
+});
+```
+
+Note: You must have verifiable ownership of the domain before customers can use private DNS. The VpcEndpointServiceDomainName
+will verify ownership for you, assuming you own the domain.
```

Comment:
might be helpful to add what `verifiable` includes. can we guide users towards any action required for `VpcEndpointServiceDomainName` to perform the verification successfully?